src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      discount = 20;
```

Comment:
The condition `days >= 3 && days < 7` can be simplified by using an `if` statement with a return inside. This way, you don't need the `break` statement and the code becomes cleaner.